_data/utilities.yml

```diff
@@ -73,9 +73,8 @@ websites:
       tfa: No
 
     - name: Dominion
-      url: https://www.dom.com/
-      twitter: DomVAPower
-      facebook: dominionvirginiapower
+      url: https://www.dominionenergy.com/
+      twitter: dominionenergy
```

Comment:
![chrome_2017-06-22_20-30-11](https://user-images.githubusercontent.com/3535780/27449722-a4c1c6f4-5789-11e7-9d35-2c985e177080.png)
There are actually a few replies from the account.